.github/workflows/unit_tests.yml

```diff
@@ -29,6 +29,11 @@ jobs:
             sudo apt-get update
             sudo apt-get install libopenmpi-dev openmpi-bin
           fi;
+      - name: Install Neuron on Windows separately
+        shell: cmd
+        run: |
+          Copy-Item -Source https://github.com/neuronsimulator/nrn/releases/download/8.2.0/nrn-8.2.0.w64-mingw-py-37-38-39-310-setup.exe -Destination .
+          start /b /wait .\nrn-8.2.0.w64-mingw-py-37-38-39-310-setup.exe /S /D=C:\nrn_test
```

Comment:
copied this from Neuron repo: https://github.com/neuronsimulator/nrn/blob/3360cd6531d3f7502c59f4d0e8ee92a0f48abe3b/ci/win_install_neuron.cmd#L2

rest is a bit of guesswork. My powershell skills are weak ... might be easier for someone with access to powershell to debug.

Reply:
I don't know any powershell and won't have much time to learn over the next week either. Maybe we can aim to pick this back up the week of the 19th-23rd?

Reply:
okay sounds good. Let's aim to release v0.3 with a working Windows install ... that would be great!